### PR TITLE
Update cli reset type flag values

### PR DESCRIPTION
**Pull request by feedmeapples**

### CODE REVIEW

File: docs/tctl.md [hunk 1]
```diff
@@ -269,9 +269,10 @@ You can reset to some predefined event types:
 ./tctl workflow reset -w <wid> -r <rid> --reset_type <reset_type> --reason "some_reason"
 ```
 
-- FirstDecisionCompleted: reset to the beginning of the history.
-- LastDecisionCompleted: reset to the end of the history.
+- FirstWorkflowTask: reset to the beginning of the history.
+- LastWorkflowTask: reset to the end of the history.
 - LastContinuedAsNew: reset to the end of the history for the previous run.
+- BadBinary: <>.
```
Comment (feedmeapples) [inline docs/tctl.md:275]: @mastermanu, @cullywakelin do you know what this resets to?

Comment (mastermanu) [reply docs/tctl.md:275]: it resets the workflow to the point where a workflowtask executed with the version of the bad binary and then starts from there

File: docs/tctl.md [hunk 2]
```diff
@@ -269,9 +269,10 @@ You can reset to some predefined event types:
...
 
 If you are familiar with the Temporal history event, You can also reset to any decision finish event by using:
```
Comment (mastermanu) [inline docs/tctl.md:277]: "decision" is used here too. Let's CTRL-F the entire file for decision and replace where necessary

File: docs/tctl.md (hunk 1 above)
Comment (feedmeapples) [inline docs/tctl.md:275]:
```suggestion
- BadBinary: reset to the point where the version of the binary would throw.
```

Comment (feedmeapples) [reply docs/tctl.md:275]: @cullywakelin does this wording look fine?

Comment (cullywakelin) [reply docs/tctl.md:275]: Maybe just:
> Reset to the point where a bad binary was used.

**Review by cullywakelin — APPROVED** [review]

**Pull request merged**